tests/cli_test.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from tests.draw_test import TestBase
+import unittest
+from worldengine import __main__
+from worldengine.cli.main import main
+
+
+class TestCLI(TestBase):
+
+    def setUp(self):
+        super(TestCLI, self).setUp()
+        self.world = "%s/seed_28070.world" % self.tests_data_dir
+
+    def test__main__(self):
+        assert __main__
+
+    def test_options(self):
```

Comment:
+1

maybe we could split this one in a few separate tests, to verify each single operation/option works (of course I am not expecting you to write tests for all these cases as part of this PR)